packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -232,6 +232,8 @@
 
 @mixin item-label() {
     @include typography.generate(form-fields-theme.$item-label-typography);
+
+    color: form-fields-theme.$label-color-default;
```

Comment:
There are different color tokens in Figma. But I believe yours are correct and Figma is wrong here because you have it synced with the form fields.

In Figma:

- text primary for label
- text secondary for helper text

Reply:
I discussed it with Jirka and he will update it in Figma